src/translations/nbTranslations.json

```diff
@@ -2038,6 +2038,11 @@
     "export": "Eksporter",
     "field_label": "Felt",
     "health_project_type": "Helseprosjekttype",
+    "health_project_types": {
+      "DRUGSTUDY": "Legemiddelstudium",
+      "OTHERCLIN": "Annet klinisk behandlingsstudium",
+      "OTHERSTUDY": "Annet studium"
```

Comment:
Disse matcher nå det som søket returnerer Cristin, derfor kan vi med fordel legge inn oversettelser for engelsk og nynorsk selv etter merge